### PR TITLE
Added vcf module in Utils, removed hardcoded extensions in markdup & …

### PR DESCRIPTION
**Pull request by sawibo**
- Added VCF finder in Utils
- Removed hardcoded extension in MarkDup en MergeBams
- Added a temporary fix in CollectWGSMetrics that renames an output field (needed for MultiQC to pick it up)

### CODE REVIEW

File: MultiQC/1.5/MultiQC.nf [hunk 1]
```diff
@@ -6,15 +6,17 @@ process MultiQC {
     container = 'library://sawibo/default/bioinf-tools:multiqc-1.5'
 
     input:
-    val id
+
```
Comment (tilschaef) [inline MultiQC/1.5/MultiQC.nf]: Would it be an idea to keep id and pass it to -f ?  We could set the title of the report based on the run id for example.

Comment (sawibo) [reply MultiQC/1.5/MultiQC.nf]: Goal was here to make this module similar to the 1.8 version and id is not really needed imo

File: fgbio/1.1.0/CallMolecularConsensusReads.nf [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    clusterOptions = workflow.profile == "sge" ? "-l h_vmem=${params.callmolecularconsensusreads.mem}" : ""
+
+    // container = 'library://sawibo/default/bioinf-tools:gatk4.1.3.0'
+
+    input:
+      tuple sample_id, file(bam)
+
+    output:
+      tuple sample_id, file("${sample_id}.u.consensus.bam")
+
```
Comment (tilschaef) [inline fgbio/1.1.0/CallMolecularConsensusReads.nf]: Hardcoded paths, should be in params.

File: fgbio/1.1.0/DemuxFastqs.nf [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    // container = 'library://sawibo/default/bioinf-tools:gatk4.1.3.0'
+
+    input:
+      tuple sample_sheet, file(fastq: "*"), read_structures
+
+    output:
+      tuple file("metrics.txt"), file("*.fastq.gz")
+
+    script:
+    fgbio_path = '/hpc/local/CentOS7/cog_bioinf/fgbio-1.1.0/fgbio-1.1.0.jar'
```
Comment (tilschaef) [inline fgbio/1.1.0/DemuxFastqs.nf]: Hardcoded paths, should be in params.

File: fgbio/1.1.0/FilterConsensusReads.nf [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    // container = 'library://sawibo/default/bioinf-tools:gatk4.1.3.0'
+
+    input:
+      tuple sample_id, file(bam)
+
+    output:
+      tuple sample_id, file("${sample_id}.u.consensus.filtered.bam")
+
+    script:
+    fgbio_path = '/hpc/local/CentOS7/cog_bioinf/fgbio-1.1.0/fgbio-1.1.0.jar'
```
Comment (tilschaef) [inline fgbio/1.1.0/FilterConsensusReads.nf]: Hardcoded paths, should be in params.

File: fgbio/1.1.0/SortBam.nf [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    // container = 'library://sawibo/default/bioinf-tools:gatk4.1.3.0'
+
+    input:
+      tuple sample_id, file(bam)
+
+    output:
+      tuple sample_id, file("${sample_id}.u.grouped.sorted.bam")
+
+    script:
+    fgbio_path = '/hpc/local/CentOS7/cog_bioinf/fgbio-1.1.0/fgbio-1.1.0.jar'
```
Comment (tilschaef) [inline fgbio/1.1.0/SortBam.nf]: Hardcoded paths, should be in params.

File: fgbio/1.1.0/DemuxFastqs.nf [hunk 6]
```diff
@@ -0,0 +1,26 @@
+process DemuxFastqs {
+    tag {"FGBIO_demuxfastqs "}
+    label 'FGBIO_1_1_0'
+    label 'FGBIO_demuxfastqs_1_1_0'
```
Comment (tilschaef) [inline fgbio/1.1.0/DemuxFastqs.nf]: Label for subcommand should be: 'Tool_verion_Command'

File: fgbio/1.1.0/FilterConsensusReads.nf [hunk 7]
```diff
@@ -0,0 +1,25 @@
+process FilterConsensusReads {
+    tag {"FGBIO_filterconsensusreads ${sample_id}"}
+    label 'FGBIO_1_1_0'
+    label 'FGBIO_filterconsensusreads_1_1_0'
```
Comment (tilschaef) [inline fgbio/1.1.0/FilterConsensusReads.nf]: Label for subcommand should be: 'Tool_verion_Command'

File: fgbio/1.1.0/SortBam.nf [hunk 8]
```diff
@@ -0,0 +1,24 @@
+process SortBam {
+    tag {"FGBIO_sortbam ${sample_id}"}
+    label 'FGBIO_1_1_0'
+    label 'FGBIO_sortbam_1_1_0'
```
Comment (tilschaef) [inline fgbio/1.1.0/SortBam.nf]: Label for subcommand should be: 'Tool_verion_Command'

File: fgbio/1.1.0/FilterConsensusReads.nf (hunk 7 above)
Comment (tilschaef) [inline fgbio/1.1.0/FilterConsensusReads.nf]: Label for subcommand should be: 'Tool_verion_Command'

File: python/MakeUmiBam.nf [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+    tuple sample_id, file(fastq: "*")
+
+    output:
+    tuple sample_id, file("${sample_id}.u.grouped.bam")
+
+
+    script:
+
+    """
+    #!/hpc/local/CentOS7/common/lang/python/2.7.10/bin/python
```
Comment (tilschaef) [inline python/MakeUmiBam.nf]: Add container with python 2.7 (if possible).

File: fgbio/1.1.0/CallMolecularConsensusReads.nf (hunk 2 above)
Comment (sawibo) [reply fgbio/1.1.0/CallMolecularConsensusReads.nf]: Totally agree, this is still under heavy development

File: python/MakeUmiBam.nf (hunk 9 above)
Comment (sawibo) [reply python/MakeUmiBam.nf]: On my todo list

**Pull request merged**